components/EditCarForm/index.js

```diff
@@ -0,0 +1,74 @@
+import { useState } from "react";
+import initialCars from "../../db/db.json";
+
+export default function EditCarForm({ car, onSubmit }) {
+  const defaultValue = { Milage: 0, Plate: "", ImageUrl: "" };
+  const [activeCar, setActiveCar] = useState({ ...defaultValue, ...car });
+
+  const groups = [
```

Comment:
Still waiting for this to be implemented :cry:

Reply:
The const groups is in the [vin].js and given to the component as a prop and the other ones are deleted, because I don't need them anymore. 
Could you take another look at it?

Reply:
Please just move the const groups ... a few lines up above the beginning of the component (function)

Reply:
Ok thank you, now i got it.